src/features/TransactionStatus/TransactionStatus.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useContext } from 'react';
+import { Input } from '@mycrypto/ui';
+import { withRouter } from 'react-router-dom';
+import isEmpty from 'lodash/isEmpty';
+import queryString from 'query-string';
+
+import { Button, NetworkSelectDropdown, ContentPanel, TxReceipt, InlineMessage } from '@components';
+import { ITxHash, NetworkId, ITxType, ITxConfig, ITxReceipt } from '@types';
+import { NetworkContext, AssetContext, StoreContext, ProviderHandler } from '@services';
+import {
+  makeTxConfigFromTransactionResponse,
+  makePendingTxReceipt,
+  makeTxConfigFromTxReceipt
+} from '@utils/transaction';
+import { noOp } from '@utils';
+import { useEffectOnce, useUpdateEffect } from '@vendor';
+import { DEFAULT_NETWORK, ROUTE_PATHS } from '@config';
+import { translateRaw } from '@translations';
+import { getTxsFromAccount } from '@services/Store/helpers';
+
+const SUPPORTED_NETWORKS: NetworkId[] = ['Ethereum', 'Ropsten', 'Goerli', 'Kovan', 'ETC'];
+
+const TransactionStatus = withRouter(({ history, match, location }) => {
```

Comment:
1. Naming: use `Tx`
2. Convention: Prefer a named function and export the wrapped function.
```
const TxStatus = ({ history, match, location }) => {}
export default withRouter(TxStatus)

```
3. TestUI please